app/javascript/components/StaffDashboard/index.js

```diff
@@ -73,7 +75,7 @@ class StaffDashboard extends React.Component {
                     <th>PAPERWORK</th>
                     <th>CASE NOTES</th>
                     <th>FORM STATUS</th>
-                    <th> </th>
+                    <th></th>
```

Comment:
@didvi do you remember why this empty tag is here? When i remove it no UI changes as far as I can tell. Is it safe to delete?